assets/models/organelles/Cilia.tscn

```diff
@@ -1,102 +1,57 @@
-
-[node name="Cube_001" parent="Armature_001/Skeleton3D" index="0"]
-material_override = SubResource("ShaderMaterial_axf8n")
-
-[node name="Skeleton3D" parent="Armature_002" index="0"]
-bones/0/rotation = Quaternion(0.476189, 0.522727, -0.522727, 0.476189)
-bones/2/rotation = Quaternion(7.419e-08, -1.66529e-08, 0.126457, 0.991972)
-bones/2/scale = Vector3(1, 1, 1)
-bones/3/rotation = Quaternion(9.24412e-15, 2.1946e-08, 1.11759e-08, 1)
-bones/4/rotation = Quaternion(0.040004, 0.000781606, 0.0195197, 0.999009)
-bones/4/scale = Vector3(1, 1, 1)
-bones/5/rotation = Quaternion(5.39554e-09, 0.000422143, 0.00526191, 0.999986)
-bones/6/rotation = Quaternion(-0.0183167, -0.00669896, -0.108578, 0.993897)
-bones/7/rotation = Quaternion(1.7993e-08, 5.6693e-08, 6.58101e-08, 1)
-
-[node name="Cube_002" parent="Armature_002/Skeleton3D" index="0"]
-material_override = SubResource("ShaderMaterial_w2p6n")
-
-[node name="AnimationPlayer" parent="." index="3"]
+bones/1/rotation = Quaternion(-0.00878315, -0.0863505, 0.000288114, 0.996226)
+bones/2/rotation = Quaternion(-0.0133134, -0.0784489, -0.00723503, 0.996803)
+bones/3/rotation = Quaternion(0.0178515, 0.211634, 0.106178, 0.9714)
+bones/4/rotation = Quaternion(-0.0323654, 0.280138, 0.119343, 0.951962)
+bones/5/rotation = Quaternion(0.0323044, -0.122566, -0.109227, 0.985902)
+bones/6/rotation = Quaternion(0.0590854, -0.139002, -0.169496, 0.973888)
+bones/7/rotation = Quaternion(-0.009155, -0.14818, -0.163101, 0.975375)
+bones/8/rotation = Quaternion(0.0266372, 0.103122, 0.0486732, 0.99312)
+bones/9/rotation = Quaternion(0.0567211, 0.662795, 0.0371481, 0.745725)
+bones/10/rotation = Quaternion(-0.00127555, 0.220357, 0.0398706, 0.974603)
+bones/11/rotation = Quaternion(-0.0232308, -0.119849, 0.0612927, 0.990626)
+bones/12/rotation = Quaternion(0.00409056, -0.163622, 0.126868, 0.978323)
+bones/13/rotation = Quaternion(0.00815998, 0.176283, 0.111464, 0.977974)
+bones/14/rotation = Quaternion(-0.026875, 0.135833, 0.0545623, 0.988863)
+bones/15/rotation = Quaternion(-0.0237881, 0.195296, 0.0460238, 0.979375)
+bones/16/rotation = Quaternion(-0.0373953, -0.061726, 0.0590191, 0.995645)
+bones/17/rotation = Quaternion(-0.00056857, 0.708348, -0.000570359, 0.705863)
+bones/18/rotation = Quaternion(0.0466885, 0.135809, -0.02992, 0.989182)
+bones/19/rotation = Quaternion(-0.00475943, -0.109584, -0.0167473, 0.993825)
+bones/20/rotation = Quaternion(0.0323454, -0.220061, 0.13441, 0.96564)
+bones/21/rotation = Quaternion(0.0379198, 0.0480719, 0.0459524, 0.997066)
+bones/22/rotation = Quaternion(-0.035468, 0.236547, -0.112789, 0.964399)
+bones/23/rotation = Quaternion(0.0341753, 0.119834, -0.0883902, 0.988261)
+bones/24/rotation = Quaternion(-0.0142192, -0.114817, 0.051429, 0.991953)
+
+[node name="CiliaMesh" parent="Armature/Skeleton3D" index="0"]
+material_override = SubResource("ShaderMaterial_jjmap")
+mesh = ExtResource("5_uvfp1")
+
+[node name="AnimationPlayer" parent="." index="1"]
 autoplay = "wiggle"
-speed_scale = 0.2
```

Comment:
This also has the changed speed scale like flagellum. I think you fixed that there, but this is still like this...